pytest_tests/lib/helpers/aws_cli_client.py

```diff
@@ -380,20 +361,7 @@ def delete_bucket_tagging(self, Bucket: str) -> dict:
         output = _cmd_run(cmd)
         return self._to_json(output)
 
-    def put_object_retention(
```

Comment:
This one as well.

Reply:
This one was also shown by the linter, there were two methods with the same signatures. I will move it to a separate commit.